auto-dark.el

```diff
@@ -26,6 +26,8 @@
 
```

Comment:
If you could please bump the package version to:
`;; Version: 0.9`
This is for sure the next feature to be merged to master.